### PR TITLE
IR-8736 unit tests gltf loading images

### PR DESCRIPTION
**Pull request by quaiquai-ir**
## References
https://tsu.atlassian.net/browse/IR-8736

## QA Steps
npx vitest --config=../../vitest.client.config.ts loader/GLTFLoader.image.test.tsx

### CODE REVIEW

File: packages/engine/src/gltf/loader/GLTFLoader.image.test.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ CPAL-1.0 License
 The contents of this file are subject to the Common Public Attribution License
 Version 1.0. (the "License"); you may not use this file except in compliance
 with the License. You may obtain a copy of the License at
-https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
+https:/github.com/ir-engine/ir-engine/blob/dev/LICENSE.
```
Comment (HexaField) [inline packages/engine/src/gltf/loader/GLTFLoader.image.test.tsx]: odd deleted forward slash??? lol

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/engine/src/gltf/loader/GLTFLoader.image.test.tsx (hunk 1 above)
Comment (quaiquai-ir) [reply packages/engine/src/gltf/loader/GLTFLoader.image.test.tsx]: ... ugh the side effect of deleting AI comments and missing the edge case LOL

**Review by HexaField — APPROVED** [review]
